src/bugjira/bugzilla.py

```diff
@@ -0,0 +1,37 @@
+from bugzilla import Bugzilla as BzBugzilla
+
+from bugjira.broker import Broker
+from bugjira.config import get_config
+from bugjira.dao import Issue
+
+
+class Bugzilla(Broker):
```

Comment:
took me a moment to see difference and naming overlap in bugjira.py Bugzilla is the class from bugzilla but in there the same class is BzBugzilla and this class overlaps.